### PR TITLE
INTGR: feat/Implement Footer component

### PR DESCRIPTION
**Pull request by williamwgeotab**
- Add initial footer component content
- Add styling matching figma
- Add logo and divider svg
- Add footer component to App.tsx

### CODE REVIEW

File: src/components/Footer/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import { FooterDivider, LogoGeotabFooter } from "../Footer";
+import "./footer.scss";
+
+export default function Footer() {
+  return (
+    <div>
+      <footer>
+        <div className="footer-container">
+          <div className="footer-container__left">
+            <div className="footer-logo-container">
```
Comment (kmets89) [inline src/components/Footer/Footer.tsx:10]: looks like this class doesn't exist?

File: src/components/Footer/footer.scss [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+.footer-container__left {
+  display: flex;
+  align-items: center;
+  gap: 24px;
+
+  a {
+    color: $backgrounds-content-2;
+    font-family: Roboto;
```
Comment (kmets89) [inline src/components/Footer/footer.scss:19]: the font family is set in project level css file, please don't introduce redundant styling

**Review by kmets89 — CHANGES_REQUESTED** [review]

File: src/components/Footer/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
+import { FooterDivider, LogoGeotabFooter } from "../Footer";
+import "./footer.scss";
+
+export default function Footer() {
+  return (
+    <div>
+      <footer>
+        <div className="footer-container">
+          <div className="footer-container__left">
```
[the remark this answers is not in the document]
Comment (williamwgeotab) [reply src/components/Footer/Footer.tsx:9]: fixed, ty!

**Comment by williamwgeotab** [comment]
@kmets89 merged most recent changes of INTGR-123 & resolved the outstanding items. Thanks!

**Review by kmets89 — APPROVED** [review]

**Pull request merged**